### PR TITLE
Removed chainlink from oracle-starter-kit

### PR DESCRIPTION
**Pull request by praveen-klaytn**
## Proposed changes

- Removed chainlink code snippets from oracle-starter-kit .

## Types of changes

Please put an x in the boxes related to your change.

- [X] Bugfix
- [ ] New feature or Enhancement
- [ ] Others

## Checklist

*Put an x in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code.*

- [X] I have read the [CONTRIBUTING GUIDELINES](https://github.com/klaytn/klaytn-developer-sdk/blob/main/CONTRIBUTING.md) doc
- [X] I have signed the CLA by posting "I have read the CLA Document and I hereby sign the CLA" in the comment
- [X] Lint and unit tests pass locally with my changes
- [X] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: packages/kss-cli/bin/kds-cli.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ program.version(packageData.version)
 
 // Commands
 program
-.name('kds-cli')
```
Comment (toniya-klaytn) [inline packages/kss-cli/bin/kds-cli.js:13]: @praveen-klaytn need to rename this file

**Review by toniya-klaytn — COMMENTED** [review]
https://github.com/klaytn/klaytn-developer-sdk/pull/34/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519L15 
is this still relevant ?

**Review by toniya-klaytn — COMMENTED** [review]
https://github.com/praveen-klaytn/klaytn-developer-sdk/blob/main/packages/oracles-starter-kit/README.md#1-request-new-randomness

replace 
  npx hardhat request-witnet-randomness --contract 0x8937C127F3060fF8a23E9a0fb5AEA10bc30e28be --network baobab
with 
npx hardhat request-witnet-randomness --contract 0x39D1beEBD3647DCE4DAD97a6223fb7b48b6Cf64f --network baobab --value 1

**Comment by praveen-klaytn** [comment]
_**I have read the CLA Document and I hereby sign the CLA**_

**Comment by praveen-klaytn** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by praveen-klaytn** [comment]
recheck

**Comment by Sotatek-TinnNguyen** [comment]
recheck

**Comment by praveen-klaytn** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by praveen-klaytn** [comment]
recheck

**Pull request closed without merging**